coalib/bears/requirements/NpmRequirement.py

```diff
@@ -24,3 +25,34 @@ def __init__(self, package, version=""):
         :param version: A version string. Leave empty to specify latest version.
         """
         PackageRequirement.__init__(self, 'npm', package, version)
+
+    def install_command(self):
+        """
+        Creates the installation command for the instance of the class.
+
+        >>> pr = NpmRequirement('alex', '2')
+        >>> pr.install_command()
+        'npm install alex@2'
+
+        :param return: A string with the installation command.
+        """
+        return "npm install {}@{}".format(self.package, self.version)
+
+    def is_installed(self):
+        """
+        Checks if the dependency is installed.
+
+        >>> pr = NpmRequirement('npm')
+        >>> pr.is_installed()
+        True
+
+        >>> pr = NpmRequirement('some_package')
+        >>> pr.is_installed()
+        False
+
+        :param return: True if dependency is installed, false otherwise.
+        """
+        if shutil.which(self.package):
```

Comment:
i dont know how.. i havent found a way to ask npm to show me if its installed..ideas?